workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/CommandDiffDeserializerGenerator.tt

```diff
@@ -0,0 +1,70 @@
+<#@ template language="C#" #>
+<#@ output extension=".cs" #>
+<#
+    var componentDetails = GetComponentDetails();
+    var generatedHeader = CommonGeneratorUtils.GetGeneratedHeader();
+    var commandDetailsList = GetCommandDetailsList();
+#>
+<#= generatedHeader #>
+
+using Unity.Mathematics;
+using Unity.Entities;
+using Improbable.Gdk.Core;
+using Improbable.Worker.CInterop;
+
+namespace <#= qualifiedNamespace #>
+{
+    public partial class <#= componentDetails.ComponentName #>
+    {
+<# foreach (var command in commandDetailsList) {
+        var receivedRequestType = command.CommandName + ".ReceivedRequest";
+        var receivedResponseType = command.CommandName + ".RawReceivedResponse";
+#>
+        public class <#= command.CommandName #>DiffCommandDeserializer : ICommandDiffDeserializer
+        {
+            public uint GetComponentId()
+            {
+                return ComponentId;
```

Comment:
Is this declared in the `partial` parent class?

Reply:
Yes